assets/src/styles/components/_PageHistory.scss

```diff
@@ -0,0 +1,238 @@
+$border-color-light-grey: darken( $border-color, 10% );
+$mq-lg-up:   "screen and ( min-width: 70rem )";
+$mq-lg-down: "screen and ( max-width: 70rem )";
+
+.PageHistory_Diff-Added {
+	background: #daf2d6;
+	text-decoration: none;
+}
+
+.PageHistory_Diff-Removed {
+	background: lighten( $hm-primary, 35% );;
```

Comment:
Double semicolon here too.. these styles are a duplicate of the file above, can we use one style for both instances?